engine/battle_anims/bg_effects.asm

```diff
@@ -2589,8 +2589,8 @@ BGEffects_LoadBGPal1_OBPal0:
 	ld a, h
 	push bc
 	push af
-	ld hl, wBGPals2 palette 1
-	ld de, wBGPals1 palette 1
+	ld hl, wBGPals2 palette PAL_BATTLE_BG_ENEMY
```

Comment:
`ld hl, wOBPals2` and `ld de, wOBPals1` above should have `palette PAL_BATTLE_OB_ENEMY`, and the whole function should be renamed to `BGEffects_LoadEnemyPals`.